macros/utils/cross_db_utils/get_query_history.sql

```diff
@@ -0,0 +1,31 @@
+{% macro get_query_history(from_time) %}
+  {% do return(adapter.dispatch("get_query_history", "elementary")(from_time)) %}
+{% endmacro %}
+
+{% macro bigquery__get_query_history(from_time) %}
+select 
+  job_id || referenced_table.project_id || referenced_table.dataset_id || referenced_table.table_id as id,
+  job_id as job_id,
+  statement_type,
+  start_time,
+  end_time,
+  state,
+  referenced_table.project_id as source_db,
+  referenced_table.dataset_id as source_schema,
+  referenced_table.table_id as source_table,
+  destination_table.project_id as dest_db,
+  destination_table.dataset_id as dest_schema,
+  destination_table.table_id as dest_table
+from `{{ target.project }}`.`region-{{ target.location }}`.INFORMATION_SCHEMA.JOBS, 
```

Comment:
Why don't you use `.information_schema()` like [here](https://github.com/elementary-data/dbt-data-reliability/blob/618d592ad91ff5ca61dc942036edb8cffa95b0f6/macros/edr/metadata_collection/get_columns_from_information_schema.sql#L22)?

Reply:
honestly did not think about it, tho it might not work due to the region as part of the identifier, will check